core/tests/Test.Microsoft.Identity.LabInfrastructure/LabServiceApi.cs

```diff
@@ -91,17 +93,18 @@ private IUser GetUserFromLab(UserQueryParameters query)
                 user.InitializeHomeUser();
 
             user.KeyVault = _keyVault;
-            return user;
+            return response;
         }
 
         /// <summary>
         /// Returns a test user account for use in testing.
         /// </summary>
         /// <param name="query">Any and all parameters that the returned user should satisfy.</param>
         /// <returns>Users that match the given query parameters.</returns>
-        public IUser GetUser(UserQueryParameters query)
+        public ILabResponse GetUser(UserQueryParameters query)
         {
-            var user = GetUserFromLab(query) as LabUser;
+            var response = GetUserFromLab(query);
+            var user = response.User as LabUser;
```

Comment:
> response.User as LabUser [](start = 22, length = 25)

Consider making the response return the type as LabUser instead, any concerns about that?